app/javascript/src/components/InviteUsersModal/index.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Modal, ButtonToolbar, Button } from 'react-bootstrap';
+import styled from 'styled-components';
+import axios from '../../config/axios';
+
+import { INVITE_USERS_PATH, TEAM_DETAILS_PATH } from '../../config/api_endpoints';
+import InviteUsersForm from './components/InviteUsersForm';
+import InviteUsersResults from './components/InviteUsersResults';
+import InviteUsersButton from './components/InviteUsersButton';
+
+const StyledButtonToolbar = styled(ButtonToolbar)`
+	float: right;
+`;
+
+class InviteUsersModal extends Component {
+	constructor() {
```

Comment:
you have to pass "props2 in the constructor as param and then pass it to super

Reply:
:+1: